BackEnd/src/external-api/player.ts

```diff
@@ -17,6 +17,18 @@ export async function GetRiotPlayerBySummonerId(summonerId: string): Promise<Rio
   return await MakeRiotAPICall<RiotSummonerDto>(`/lol/summoner/v4/summoners/${encodeURIComponent(summonerId)}`, 'GET');
 }
 
+export async function GetRiotAccountByGameNameAndTagline(gameName: string, tagline: string): Promise<RiotAccountDto> {
+  return await MakeRiotAPICall<RiotAccountDto>(`/riot/account/v1/accounts/by-riot-id/${encodeURIComponent(gameName)}/${encodeURIComponent(tagline)}`, 'GET');
+}
+
+export async function GetRiotPlayerByGameNameAndTagline(gameName: string, tagline: string): Promise<RiotSummonerDto> {
+  try {
+    let riotAccount: RiotAccountDto = await GetRiotAccountByGameNameAndTagline(gameName, tagline);
+    return await GetRiotPlayerByPuuid(riotAccount.puuid);
+  } catch(err) {
+    return null;
+  }
```

Comment:
I think we should just throw here. We usually catch at the top level on the api calls to return a generic 500. Here tho the error at top level won't get logged.

Reply:
sure, will adjust